lib/evss/documents_service.rb

```diff
@@ -1,26 +1,62 @@
 # frozen_string_literal: true
+require 'common/exceptions'
 require_dependency 'evss/base_service'
 
 module EVSS
+  # DocumentData is a struct class bundling together the required attributes of a file upload
+  DocumentData = Struct.new(
```

Comment:
`lib` tends to be for classes that aren't necessarily part of the core app itself, like API clients and utilities that could be extracted. In this case, you could leave it in `lib` as it could be considered part of the EVSS API. I was actually thinking a bit further ahead and considering mounting the Carrierwave uploader in this class, but that could be done later. I also almost always prefer new files unless that class is only used within the file containing it.